kurtosis-devnet/justfile

```diff
@@ -82,6 +82,13 @@ devnet TEMPLATE_FILE DATA_FILE="" NAME="" PACKAGE=KURTOSIS_PACKAGE:
         fi
     fi
     export ENCL_NAME="$DEVNET_NAME"-devnet
+
+    if [[ -z "$SKIP_ENCL_CLEAN" ]]; then
```

Comment:
For consistency (with KURTOSIS_PACKAGE for example) I'd rather use a just variable (potentially initialized from the environment) than an environment variable